tests/native/test_cpu_manual.py

```diff
@@ -170,31 +150,6 @@ class SymCPUTest(unittest.TestCase):
         "IF": 0x00200,
     }
 
-    class ROOperand:
-        """ Mocking class for operand ronly """
-
-        def __init__(self, size, value):
-            self.size = size
-            self.value = value
-
-        def read(self):
-            return self.value & ((1 << self.size) - 1)
-
-    class RWOperand(ROOperand):
-        """ Mocking class for operand rw """
-
-        def write(self, value):
-            self.value = value & ((1 << self.size) - 1)
-            return self.value
-
-    def setUp(self):
-        mem = MockMemory()
-        self.cpu = I386Cpu(mem)  # TODO reset cpu in between tests...
-        # TODO mock getchar/putchar in case the instruction accesses memory directly
-
-    def tearDown(self):
-        self.cpu = None
-
```

Comment:
Note: the `self.cpu` parameter was only used in a few tests in this file previously.  So instead of creating & destroying it around each test, I changed the code to explicitly create a `Cpu` object only within the test cases that use it.